src/web-exploitation/intro-to-sqli/README.md

```diff
@@ -17,40 +17,81 @@ SQL Injection (SQLi) is a security vulnerability that allows attackers to inject
 7. [Second Order Injections](second-order-injections.md)
 
 ## Basis to Tackle SQLi Challenges
-
-* The `--` comments out the rest of the query, effectively bypassing the password check and logging the attacker in as any user.
-* Basic SQL syntax
-  1. `SELECT` Extracts data from the given table.
-  2. `ORDER BY` Orders the results gotten from SELECT in a specific manner. For example, if one has a table of countries and their populations. One can select the countries starting with the letter R and then order them by their population.
-  3. `UNION` clause allows you to group multiple queries together.
-  4. `JOIN` Joins data from two tables depending on a certain characteristic on the table. So if a theres two table one with customer IDs and their addresses and another table with customer IDs and their purchases. You can join both tables so that the customer addresses match their purchases.
-  5. `DELETE, INSERT` Allows you to delete data or add new data to a table.
-  6. `AND,OR` Allows you to modify queries so that they return information depending on multiple categories.
-  7. `MIN,MAX` They return the smallest or largest value of a query
 * Look for places from where data might be used to construct a query. Like a textbox, or the url of the website.
```

Comment:
```suggestion
* User input from textboxes, URL parameters, etc may be used to construct a SQL query in the backend
```
add this point at the end and it'd be nice to have a small example to improve clarity and explain what happens in the backend at a high level